### DISCUSSION TITLE
2021-51: 颓废的一周

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#310] [Announcements]
# 2021-51: 颓废的一周

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2021-51/

**Comment by STRRL** [comment]
可能俺是漏了几期 blog, "Iteration" 具体是什么呀？从时间上划的吗？或者说这是个什么样子的 TODO Things 管理模型？🙈

**Reply by Xuanwo** [reply]
Iteration 是 Github Projects 提供的一个简单的 TODO 管理机制，一般是两周一次，有点像 Sprint～

![image](https://user-images.githubusercontent.com/5351546/147455911-4881efd6-bc65-4716-8396-ea1be1104c86.png)

**Comment by pymongo** [comment]
暗黑3 25 赛季的武僧这么强，能比猎魔人多打 5-10 层，咋不玩武僧?

**Reply by Xuanwo** [reply]
😭😭😭，我这样的老年人真的玩不来神龙火幻身

**Reply by STRRL** [reply]
(琉璃有个奥吉德黄道老年人火幻身冲层, 可以了解一下

**Comment by Endle** [comment]
如果连续数个 Iteration 完成的不好，会被扣工资吗（笑）

**Reply by Xuanwo** [reply]
据我所知我的老板是不看这个的（
